core/src/docs/upgrade.md

```diff
@@ -1,3 +1,15 @@
+# Unreleased
+
+## Public API
+
+### RFC-2774 Lister API
+
+RFC-2774 proposes a new `lister` API to replace current `list` and `scan`. And we add a new API `list` to return entries directly.
```

Comment:
`scan` removed? RFC did not mention this.

Reply:
Yes, there is an overlap between `scan` and `list_with`, which could cause issues when using `scan_with("path").delimiter("")`.